app/views/layouts/comfy/admin/cms.html.haml

```diff
@@ -1,7 +1,7 @@
 !!!
 %html{lang: I18n.locale}
   = stylesheet_link_tag 'application', media: 'all', 'data-turbo-track': 'reload'
-  = javascript_pack_tag 'trix'
+
```

Comment:
⚠️  @Pralish @alis-khadka this causes email composition to break
<img width="1728" alt="Screen Shot 2023-09-20 at 3 40 18 PM" src="https://github.com/restarone/violet_rails/assets/35935196/d6924f49-2f26-4654-bc92-2d1bb17ada18">